resources/views/components/profile.blade.php

```diff
@@ -0,0 +1,54 @@
+@props([
+    'user',
+    'buttons',
+    'stats' => [
+        'lists' => 0,
+        'reviews' => 0,
+        'friends' => 0,
+    ],
+    
+])
+
+<div class="relative flex w-full">
+    <div class="absolute top-0 right-0">
+        <x-lucide-ellipsis-vertical class="h-6 w-6 text-slate-50" />
+    </div>
+
+    <div class="mr-6 flex w-full flex-row flex-wrap items-center gap-4">
+        <div class="flex size-30 flex-none items-center justify-center">
+            @isset($user)
+                <x-avatar :image="$user->image" size="lg" />
+            @else
+                <x-avatar size="lg" />
+            @endisset
+        </div>
```

Comment:
i don't think there needs to be a wrapping div for the avatar. there's also no need to check if there is a `$user` because this will never be used where there potentially isn't a user. `x-avatar` handles if the user doesn't have a profile picture

```suggestion
                <x-avatar :image="$user->image" size="lg" />
```

Reply:
Är wrapper onödig tar vi bort den. Check finns för att inte krasha i test

Reply:
vilka test? denna komponent används bara på profilsidan och man kan inte gå in på en profil som inte existerar. då kommer man komma till 404 innan denna sida renderar

Reply:
fixad